typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -22,9 +30,9 @@ import {
 } from './multisend';
 
 export enum SubmissionType {
-  MANUAL = 'MANUAL',
-  SIGNER = 'SIGNER',
-  SAFE = 'SAFE',
+  MANUAL = 0,
+  SAFE = 1,
+  SIGNER = 2,
```

Comment:
why change this?

Reply:
reordering seems especially odd

Reply:
I wanted signer and safe to be truthy but I can revert this if you're comfortable with it